infra/firebase.compose.json

```diff
@@ -13,9 +13,9 @@
     },
     {
       "predeploy": [
-        "python -m venv llm/venv",
+        "python3 -m venv llm/venv",
```

Comment:
On my Ubuntu machine, `python` doesn't actually exist. I am pretty sure Ubuntu decided people should use the `3` suffix a long time ago.